src/utils/save_game/save_game.cpp

```diff
@@ -0,0 +1,66 @@
+#include "save_game.h"
+#include "../../creature/creature.h"
+#include "../../creature/main_character/main_character.h"
+#include <iostream>
+#include <fstream>
+
+void SaveGame(MainCharacter& main_characher, Creature& creature, const std::string& filename)
+{
+	std::ofstream outputFile(filename);
+
+	if (outputFile.is_open())
+	{
+		outputFile << "Ability Scores:" << ::std::endl;
+		outputFile << "==============================" << std::endl;
+		outputFile << "HP: " << creature.get_hp() << std::endl;
+		outputFile << "AC: " << creature.get_ac() << std::endl;
+		outputFile << "Speed: " << creature.get_speed() << std::endl;
+		outputFile << "Strength: " << creature.get_ability_score(0) << std::endl;
+		outputFile << "Dexterity: " << creature.get_ability_score(1) << std::endl;
+		outputFile << "Constitution: " << creature.get_ability_score(2) << std::endl;
+		outputFile << "Intelligence: " << creature.get_ability_score(3) << std::endl;
+		outputFile << "Wisdom: " << creature.get_ability_score(4) << std::endl;
+		outputFile << "Charisma: " << creature.get_ability_score(5) << std::endl;
+		outputFile << "==============================" << std::endl;
+		outputFile << "Ability Scores for: " << main_characher.get_name() << std::endl;
+		outputFile << "==============================" << std::endl;
+		outputFile << "HP: " << main_characher.get_hp() << std::endl;
```

Comment:
Have you given thought to how these ability scores can be easily extracted later? Will we have trouble reading from this file down the line?